packages/react-devtools-shared/src/utils.js

```diff
@@ -948,7 +948,7 @@ export function formatDataForPreview(
     case 'date':
       return data.toString();
     case 'class_instance':
-      return data.constructor.name;
+      return Object.getPrototypeOf(data).constructor.name;
```

Comment:
I would rather change it to something like this:
```
let resolvedConstructorName = data.constructor.name;
if (typeof resolvedConstructorName === 'string') {
  return resolvedConstructorName;
}

resolvedConstructorName =  Object.getPrototypeOf(data).constructor.name;
if (typeof resolvedConstructorName === 'string') {
  return resolvedConstructorName;
}

try {
  return truncateForDisplay(String(data));
} catch (error) {
  return 'unserializable';
}
```

What do you think?

Reply:
I have a feeling that other stuff should be inside the `try` block as well, but other than that it looks good to me :) Do you have push access to my branch ("Allow edits by maintainers" is checked) or would you like me to make my own commit for this?

Reply:
Could you please push it? You can also wrap everything in try block, yeah